parm/config/config.resources

```diff
@@ -156,6 +157,43 @@ elif [ $step = "atmanalpost" ]; then
     export nth_atmanalpost=1
     export npe_node_atmanalpost=$(echo "$npe_node_max / $nth_atmanalpost" | bc)
 
+elif [ $step = "aeroanlinit" ]; then
```

Comment:
are we not going to run this as part of `CDUMP = gfs`?
i.e. Is there no need for the `_gfs` counterparts?

Reply:
I think exact structure is still TBD. My guess is we will *only* run it as part of the early cycle to initialize GEFS. However, at this phase it's just to cycle so I would run GDAS only in the workflow.I suppose it's easy enough to add `_gfs` here for now just to not paint ourselves into a corner.

Reply:
Understood. 
They don't need to be added right now if there is no use case, but if someone tries to configure with _gfs, a warning could/should be added.

Reply:
I've updated it to match what is there for the JEDI ATM work. Let me know if there are additional `gfs` specific things that need added.